src/azure-cli/azure/cli/command_modules/security/_help.py

```diff
@@ -514,11 +514,12 @@
 short-summary: Updates the Azure defender plan for the subscription.
 examples:
   - name: Updates the Azure defender plan for the subscription.
+    command: >
+        az security pricing create -n VirtualMachines --tier standard
+
+  - name: Updates the Azure defender plan for the subscription with suplan and extensions
     text: >
-        az security pricing create -n VirtualMachines --tier 'standard'
-  - name: Updates the Azure defender plan for the subscription. (autogenerated)
-    text: az security pricing create -n VirtualMachines --tier 'standard'
-    crafted: true
+        az security pricing create -n VirtualMachines --tier standard --subplan P2 --extensions name=MdeDesignatedSubscription isEnabled=False --extensions name=AgentlessVmScanning isEnabled=True additionalExtensionProperties='{ \\\"ExclusionTags\\\": \\\"[{\\\\\\\"key\\\\\\\":\\\\\\\"A\\\\\\\",\\\\\\\"value\\\\\\\":\\\\\\\"B\\\\\\\"}]\\\" }'
```

Comment:
It seems the example is invalid. Why so many escape characters?

Reply:
To output the required format to be used for the input, when it is printed it is like this
<img width="620" alt="image" src="https://github.com/Azure/azure-cli/assets/139862870/5318d3d1-7634-4372-b702-f735141e43b2">

Any idea how to pass this stage since the example is valid

Reply:
When the input in the example reaches the command it is like this

"ExclusionTags": "[{\"key\":\"A\",\"value\":\"B\"]"

The slashes are required for the escaping